### PR TITLE
Started /buildpost command

### PR DESCRIPTION
**Pull request by BlockOG**

### CODE REVIEW

File: src/main/java/muriplz/kryeittpplugin/KryeitTPPlugin.java [hunk 1]
```diff
@@ -3,6 +3,7 @@
 import muriplz.kryeittpplugin.commands.*;
 import org.bukkit.Bukkit;
 import org.bukkit.ChatColor;
+import org.bukkit.command.TabCompleter;
```
Comment (muriplz) [inline src/main/java/muriplz/kryeittpplugin/KryeitTPPlugin.java]: This has no use right? there is an option to optimize all imports from all classes

Comment (BlockOG) [reply src/main/java/muriplz/kryeittpplugin/KryeitTPPlugin.java]: Yeah it has no use Idk why it's there

**Comment by BlockOG** [comment]
Should I make it so the protection and anti fall damage automatically get put?

**Comment by muriplz** [comment]
> Should I make it so the protection and anti fall damage automatically get put?

that's other thing to worry, but we will leave it for the future, the fall damage im almost there to have it working. I see everything should work, imma push this into the main branch and i am gonna test it

**Pull request merged**